.gitignore

```diff
@@ -42,3 +42,7 @@ lib/libesp32/berry/berry
 .vscode/extensions.json
 *.bak
 *.code-workspace
+
+## Python virtual environments for Platformio ##
+venv*
```

Comment:
Shouldn't it be:
```
venv
.venv
```

This seems a little too aggressive

Reply:
fixed